src/theme/typography.ts

```diff
@@ -72,12 +74,34 @@ const margin: Margin = {
   },
 };
 
+const links: Links = {
+  primary: {
+    color: colors.blueberryClassic,
```

Comment:
even though `primary` and `danger` use the same colors, i didn't want to reuse `button` colors since it would increase coupling between the files. imports between files of the same level of file hierarchy are dangerous and can create spaghetti code (unless there's some reasonable underlying semantics - `typography -> import from 'buttons'` = NONO, but `typography -> import from 'color-base' and buttons -> mport from 'color-base'` could work, if we really want to 'deduplicate' this).
what do you think @ajkl2533?

Reply:
OK leave it this way for now 👍